assets/dev/js/frontend/handlers/container/container.js

```diff
@@ -1,11 +1,5 @@
 export default [
-	() => import( /* webpackChunkName: 'container' */ '../background-slideshow' ),
-	() => import( /* webpackChunkName: 'container' */ '../background-video' ),
 	() => import( /* webpackChunkName: 'container' */ './handles-position' ),
```

Comment:
It looks like the `handles-position` handler  is still dupicated

Reply:
It's not, there are 2 `handles-position` files. One for section and one for Container.